lib/src/barback/dartdevc/module_config_transformer.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Whoops I meant to exclude this file and add it as a followup - I will do that tomorrow so please ignore for now.